sysdig/resource_sysdig_secure_cloud_auth_account.go

```diff
@@ -130,11 +130,11 @@ func resourceSysdigSecureCloudauthAccount() *schema.Resource {
 				Optional: true,
 				Computed: true,
 			},
-			"cloud_provider_id": {
+			"provider_id": {
```

Comment:
do you want to have them defined as constants above?

Reply:
Keeping with the convention in the provider framework as other resources, I think its fine. Besides do you see any value in defining them as constants?

Reply:
well sysdig/common.go technically should have these, but i don't see all of the resources followed the convention. Upto you

Reply:
Ah I see. Will take a look and update the PR.

Reply:
Done.